apps/dashboard/src/components/subscribers/subscriber-row.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from '@/components/primitives/dropdown-menu';
+import { Skeleton } from '@/components/primitives/skeleton';
+import { TableCell, TableRow } from '@/components/primitives/table';
+import { TimeDisplayHoverCard } from '@/components/time-display-hover-card';
+import TruncatedText from '@/components/truncated-text';
+import { useEnvironment } from '@/context/environment/hooks';
+import { formatDate } from '@/utils/format-date';
+import { buildRoute, ROUTES } from '@/utils/routes';
+import { cn } from '@/utils/ui';
+import { ISubscriberResponseDto } from '@novu/shared';
+import { ComponentProps } from 'react';
+import { RiFileCopyLine, RiMore2Fill, RiPulseFill } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+import { Avatar, AvatarFallback, AvatarImage } from '../primitives/avatar';
+import { CompactButton } from '../primitives/button-compact';
+import { CopyButton } from '../primitives/copy-button';
+
+type SubscriberRowProps = {
+  subscriber: ISubscriberResponseDto;
+};
+
+type SubscriberLinkTableCellProps = ComponentProps<typeof TableCell> & {
+  subscriber: ISubscriberResponseDto;
+};
+
+const SubscriberLinkTableCell = (props: SubscriberLinkTableCellProps) => {
+  const { subscriber, children, className, ...rest } = props;
+
+  return (
+    <TableCell className={cn('group-hover:bg-neutral-alpha-50 text-foreground-600 relative', className)} {...rest}>
+      {children}
+    </TableCell>
+  );
+};
+
+export const SubscriberRow = ({ subscriber }: SubscriberRowProps) => {
+  const { currentEnvironment } = useEnvironment();
+
+  return (
+    <TableRow key={subscriber.subscriberId} className="group relative isolate">
+      <SubscriberLinkTableCell subscriber={subscriber}>
```

Comment:
each table column should have a width defined otherwise the content will dictate the size, resulting in a bad look on different pages, like on first page email col could be 100px but on the second 200px, resulting in jumping columns

Reply:
Good idea. Will add some widths.

Reply:
I think we need to see how to handle that for all our tables so I'll leave it for now.